api/handlers/api.go

```diff
@@ -49,6 +50,7 @@ func (a *App) New() *mux.Router {
 	apiCreate.Handle("/name-search", api.Middleware(http.HandlerFunc(n.NameSearchHandler))).Methods("GET")
 	apiCreate.Handle("/vehicle/{vehicle_id}", api.Middleware(http.HandlerFunc(v.VehicleByIDHandler))).Methods("GET")
 	apiCreate.Handle("/vehicles", api.Middleware(http.HandlerFunc(v.VehicleHandler))).Methods("GET")
+	apiCreate.Handle("/plate-search", api.Middleware(http.HandlerFunc(p.PlateSearchHandler))).Methods("GET")
```

Comment:
kudos: thanks for adding this new route. 🤜 

I would like to see some tests around this new route. I have a bunch of examples here: https://github.com/Linesmerrill/police-cad-api/blob/main/api/handlers/api_test.go

Reply:
Got everything done, just need to write the tests, should I make it in its own file in the handlers/search folder?

Reply:
let me see where best to put those.

Reply:
Ok just let me know

Reply:
Yeah guess i didnt add tests around the name-search route 😆 So I added some here https://github.com/Linesmerrill/police-cad-api/pull/7. Then you should be able to make a `plate_test.go` file in that `/search` folder and copy paste the test I just added.

If you have any questions, just let me know